.changeset/mise-release-age-excludes.md

```
@@ -0,0 +1,10 @@
+---
+---
+
+Exclude `@gtbuchanan/*` from mise's release-age quarantine
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)